packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/SelectBank/template.success.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { AddBankStepType, OBInstitution } from 'data/types'
+
+import {
+  BankSearchIcon,
+  BankSearchInput,
+  BankSearchWrapper,
+  BankWrapper,
+  ModalNavWithBackArrow,
+  SimpleBankRow
+} from '../../../../components'
+import {
+  LinkDispatchPropsType as _LD,
+  OwnProps as _O,
+  SuccessStateType as _SS} from '.'
```

Comment:
here we do not need aliases at all since all of those are building a new type `Props`